anyway/parsers/location_extraction.py

```diff
@@ -297,15 +301,60 @@ def extract_location_text(text):
     return text
 
 
+def get_related_waze_accident_alert(db, geo_location, newsflash):
+
+    # determine what distance (in kilometers) to look for waze accidents in, according to the newsflash's resolution
+    if newsflash.resolution in short_distance_resolutions:
+        distance = 0.3
+    elif newsflash.resolution in long_distance_resolutions:
+        distance = 5
+    else:
+
+        # unknown resolution - skip this optimization
+        return None
```

Comment:
There's a stray newline here.

Consider making `distance_resolution` a dictionary, mapping names to distances, on which you can iterate. Just a suggestion.